pkg/util/runtime/features.go

```diff
@@ -102,6 +105,7 @@ var (
 		FeatureGKEAutopilotExtendedDurationPods: false,
 		FeaturePlayerAllocationFilter:           false,
 		FeaturePlayerTracking:                   false,
+		FeaturePortRanges:                       false,
```

Comment:
There's a checklist above this on line 73 you need to follow a few more steps in, if you want to get this to Alpha:
* Add this to install/helm/agones/defaultfeaturegates.yaml
* Add to `ALPHA_FEATURE_GATES` in build/Makefile
* Add the inverse to the e2e-runner config in cloudbuild.yaml
* Add it to feature-stages.md